# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: data/content/newtab-render.js
# Commit: 7ec776d31a2c
# Full Hash: 7ec776d31a2c6e0ffc1e5ee311d63993317ea7ca
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2020-04-17 21:46:50
# Regressor Bug: 1614502
# File Overlap Count: 1
# Description:
#   Bug 1614502 - Expose utility functions for rendering Activity Stream. r=Mardak
#   
#   These will be used by a later patch inside of a ChromeWorker using
#   ReactDOMServer.
#   
# ==============================================================================

diff -r 8bf3fe111c40 -r 7ec776d31a2c browser/components/newtab/data/content/newtab-render.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/browser/components/newtab/data/content/newtab-render.js	Fri Apr 17 15:31:11 2020 +0000
@@ -0,0 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+"use strict";
+
+// exported by activity-stream.bundle.js
+window.NewtabRenderUtils.renderWithoutState();